index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>replit</title>
+  <link href="style.css" rel="stylesheet" type="text/css" />
+</head>
+
+<body>
+  <h1>Calculate Risk</h1>
+
+  <div>
+    <div>
+      <p>Number 1: <input placeholder="Enter a Number" type="number" class="input input1"></p>
```

Comment:
Make sure that the user is not able to add negative numbers and only a positive number range is expected.